docs/en_us/developers/source/accessibility.rst

```diff
@@ -0,0 +1,164 @@
+* Many people with disabilities use "`Assistive Technology <http://www.w3.org/TR/WCAG20/#atdef>`_" (like a screen reader) to interact with their computer, web browser, and web applications.
+* Anything conveyed visually (like an element's role, state or other properties) must also be conveyed programmatically so Assistive Technologies know the element's role, what state it's currently in and how users can interact with it.  *Protip:* This information often comes for free when you use native HTML5 elements.
+* Many people with mobility impairments use custom keyboards or keyboard emulators.
+* Some people rely on speech to interact with web applications and need to be able to address interactive elements.
+* Someone who cannot hear or hear well requires visible text captions for any audible content or an equivalent visual cue.
+* Users may customize their operating system, browser or web page display properties to make web applications easier for them to use (like reversing contrast, increasing font size, or removing images). 
+
+These are just a few core concepts developers need to keep in mind when developing user interfaces that work for everyone.  More information is available from the W3C's Web Accessibility Initiative's `How People with Disabilities Use the Web: Overview <http://www.w3.org/WAI/intro/people-use-web/Overview.html>`_.
+
+Make Your Images Accessible
+************************
+
+All images require a `text alternative <http://www.w3.org/TR/WCAG20/#text-altdef>`_ with the exception of any image that is purely decorative or has a text alternative adjacent to it.  Regardless of whether or not an image element requires a text alternative, *all* ``<img>`` elements require an ``alt`` attribute, even if the value of that attribute is ``""`` (NULL).  Because an author error should not prevent a screen reader user from using a web site, screen readers will expose the path/to/the/image as a last resort, in an effort to provide some useful information to the user regarding the purpose of the image.  If your image is purely decorative, or has a text equivalent immediately adjacent to it, use a NULL alt attribute i.e. ``alt=""``.
+
+Providing a *useful* text alternative is more nuanced than it sounds.  Asking yourself questions about the purpose of your image can usually help you determine what would be most useful to the user:
+
+* Is your image the only content of a link or form control?
+    * Your ``alt`` attribute should describe the destination of the link, or the action that will be performed.  A "Play" icon should have a text equivalent similar to "Play the 'Introduction to Linux' course video", not "Right pointing triangle."
+* Does your image contain text?
+    * Is the same text adjacent or near the image in the DOM?
+		* Use a "NULL" value, otherwise a screen reader is exposed to the content twice.
```

Comment:
Rather than (NULL), I might recommend (empty) as I think it's more humanized and readable.